packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -74,6 +74,9 @@ export interface Props {
 
 	existingSubdomain?: string;
 
+	/* this makes the domain picker in loading state */
+	areDependenciesLoading?: boolean;
```

Comment:
From the perspective of this component, what do you think about naming this prop `forceLoadingState`?